developers/README.md

```diff
@@ -17,6 +17,44 @@ Supported development platforms:
 * [Ubuntu](dev-environment/ubuntu.md)
 * [Apple's macOS](dev-environment/macOS.md)
 
+### Virtual Machines
+
+You can test changes to BonnyCI locally by exercising some tools defined in [hoist](www.github.com/BonnyCI/hoist). Hoist is a set of ansible playbooks that automate the deployment of a BonnyCI environment. Be sure that you have completed the steps in [Development Environment](#development-environment) before proceeding.
```

Comment:
nit: Not all of the development environment setup is needed, maybe those pages should be split into sections so that this can link directly to installing vagrant and the vagrant plugins?